doc/source/examples/sharp-immersed-boundary/3d-rbf-static-mixer/3d-rbf-static-mixer.rst

```diff
@@ -0,0 +1,371 @@
+|     :width: 400                                                                                                             |
+|     :name: Resulting composite geometry.                                                                                    |
+|                                                                                                                             |
++-----------------------------------------------------------------------------------------------------------------------------+
+
+---------------
+Parameter File
+---------------
+
+Simulation Control
+~~~~~~~~~~~~~~~~~~
+
+Although we are interested in the steady-state solution of the flow, we use ``bdf1`` time integration. The required time to reach steady state in our case is low, but solving it with a small value of the time step enables the non-linear solver to converge as complex flow patterns are difficult to capture otherwise.
+
+.. code-block:: text
+
+    subsection simulation control
+      set method      = bdf1
+      set time end    = 40e-4
+      set time step   = 1e-4
+      set output path = ./output/
+      set output name = output
+    end
+
+Physical Properties
+~~~~~~~~~~~~~~~~~~~
+
+We assume that the fluid is water, and that the length of the static mixer is in the order of :math:`150 \, \text{cm}`. Hence,  the length units are centimeters and the time units are seconds. The ``kinematic viscosity`` of water is :math:`0.01 \, \text{cm²}/\text{s}`.
+
+.. code-block:: text
+
+    subsection physical properties
+      subsection fluid 0
+        set kinematic viscosity = 0.01
+      end
+    end
+
+
+Mesh and Mesh Adaptation
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The mesh is a simple hyper rectangle, large enough to encompass the mixer with its casing and long enough to establish the flow profile upstream and downstream.
+
+
+.. code-block:: text
+
+    subsection mesh
+      set type      = dealii
+      set grid type = subdivided_hyper_rectangle
+
+      # Grid to use when solving the flow in the long version of the mixer
+      set grid arguments = 6,1,1: -150,-25,-25: 150,25,25: true
+
+      set initial refinement = 3
+    end
+
+Mesh adaptation ``type`` is set to ``kelly``, to allow adaptive refinement at the solid surface. This is necessary for simulations of this type because of the prohibitive mesh size required when only uniform refinement is used. Setting ``max refinement level = 5`` allows for two levels of adaptive refinement from the uniform ``initial refinement = 3`` defined in the ``mesh`` section. The ``frequency = 0`` ensures that no refinement occurs between time steps, as it is not necessary here.
+
+.. code-block:: text
+
+    subsection mesh adaptation
+      set type                 = kelly
+      set fraction type        = number
+      set max number elements  = 2000000
+      set max refinement level = 5
+      set min refinement level = 0
+      set frequency            = 0
+    end
```

Comment:
Not putting `set initial refinement steps = 2` was intended, because then we get the initial refinement from sharp and the "regular" initial refinement, which is too much. I agree that it is a bit confusing, I'll add a line and specify that.